src/main/java/net/fabricmc/loom/configuration/providers/minecraft/library/LibraryProcessorManager.java

```diff
@@ -119,9 +120,19 @@ public List<Library> processLibraries(List<LibraryProcessor> processors, List<Li
 			libraries = processedLibraries;
 		}
 
+		boolean lwjgl2 = false;
+		for (Library library : libraries) {
+			lwjgl2 |= library.name().startsWith("org.lwjgl.lwjgl:lwjgl:2.");
```

Comment:
I feel like we could break out of this loop if we find LWJGL2? Or am I misunderstanding its purpose?

Reply:
oops lol